### PR TITLE
@subsquid/commands

### PR DESCRIPTION
**Pull request by eldargab**
@belopash , please, test it on Windows.

@dzhelezov , please, review wording in readme and other places.

### CODE REVIEW

**Review by belopash — COMMENTED** [review]
Works for me

**Comment by belopash** [comment]
hmm, or not
```
TYPEGEN
Error: spawn squid-evm-typegen ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn squid-evm-typegen',
  path: 'squid-evm-typegen',
  spawnargs: [
    'src/abi',
    'erc721.json',
    '0xBB9bc244D798123fDe783fCc1C72d3Bb8C189413#something',
    '--multicall',
    '--clean'
  ]
}
```
can't execute typegen even in `git bash`

File: util/commands/src/run.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            })
+            args.push(...expanded)
+        }
+    }
+
+    let env = {...process.env, ...command.env}
+    extendPath(env, cwd)
+
+    return new Promise<number>((resolve, reject) => {
+        let proc = spawn(args[0], args.slice(1), {
```
Comment (belopash) [inline util/commands/src/run.ts:91]: If I replace `spawn` with `exec` or `execSync` it works

**Comment by eldargab** [comment]
@belopash , could you check it now?

**Comment by belopash** [comment]
> 

yes, works

**Pull request merged**